pandapower/plotting/collections.py

```diff
@@ -498,7 +498,8 @@ def _get_coords_from_geojson(gj_str):
             return ast.literal_eval(m)
         return None
 
-    if use_bus_geodata is False and line_geodata is None and ("geo" not in net.line.columns or net.line.geo.empty):
+    if use_bus_geodata is False and line_geodata is None and (
+            "geo" not in net.line.columns or net.line.geo.empty):
```

Comment:
`net.line.geo.empty` is only true if `net.line.empty` is true, since it the geo column contains every row of the net.line table even if no value is given. A check if no geo values are present should be done using net.line.geo.dropna().empty or similarly (see also plotting_toolbox.py:210 it uses `net.bus.geo.isnull().all()`).